### PR TITLE
Removed call to dependency conversions

### PR DESCRIPTION
**Pull request by willu47**
[Fixes #155522954](https://www.pivotaltracker.com/story/show/155522954)

### CODE REVIEW

**Comment by RoaldL** [comment]
This code was attempting a conversion at the beginning of a modelrun to
  1. Check if all definitions are in place and conversion is possible by Pint
  2. Feedback to the user what conversion factor is applied

I think this functionality is very useful, so I believe we need to come up with a way to support this for temporal-spatial conversions rather than deleting this. 

Travis is currently failing because it is expecting the modelrun to fail at the beginning when it is started with a non-defined conversion.

**Comment by willu47** [comment]
But this code is in sos_model, and called whenever the sos_model is built, not when a model run is performed. I agree it is useful to validate unit conversion, but I don't think it should happen here. Placing the check here breaks the code because to perform a conversion, you need to pass in an array of dims regions-by-intervals. By passing in 1 to the convert function in the instance where a space-time conversion is needed due to a mismatch between the source and sink resolutions, a full conversion is performed.

In summary, either obtain the coefficient directly from pint, and don't call convert, or remove this.

**Comment by willu47** [comment]
Superceded by #144
